src/OpenColorIO/GpuShaderDesc.cpp

```diff
@@ -70,12 +74,35 @@ class GpuShaderCreator::Impl
             m_functionHeader = rhs.m_functionHeader;
             m_functionBody   = rhs.m_functionBody;
             m_functionFooter = rhs.m_functionFooter;
+            
+            m_classWrappingInterface = CreateClassWrapper(m_language);
+            *m_classWrappingInterface = *rhs.m_classWrappingInterface;
```

Comment:
A self-explanatory writing is in fact `m_classWrappingInterface = rhs.m_classWrappingInterface->clone()`.

It becomes much more explicit and it avoids the overloading issue present with the assignation operator (i.e. what assignation operator is used when the variable is a `GpuShaderClassWrapper` type but the instance is a `MetalShaderClassWrapper` type).

Reply:
@hodoulp Thanks for the comment. The change is applied. Let me know if I can make any improvement somewhere else.